### DISCUSSION TITLE
Welcome to Virtual-ZPL-Printer Discussions!

### DISCUSSION THREAD

**Discussion by porrey** [porrey/Virtual-ZPL-Printer#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robwaim** [comment]
This is just what I was looking for.  Well done!

**Reply by porrey** [reply]
Thanks! Glad it works for you.
